CHANGELOG.rst

```diff
@@ -19,6 +19,9 @@ Fixed
 -----
 * Fixed ``discovered_devices_and_advertisement_data`` returning devices that should
   be filtered out by service UUIDs. Fixes #1576.
+* Fixed a ``Descriptor None was not found!`` exception occurring in ``start_notify()`` on Android by adjusting
+  the ``CLIENT_CHARACTERISTIC_CONFIGURATION_UUID`` to 128 bit plus the Bluetooth base UUID. Fixes #822 and 
+  #823.
```

Comment:
```suggestion
* Fixed a ``Descriptor None was not found!`` exception occurring in ``start_notify()`` on Android. Fixes  #823.
```

We can just say what was fixed and leave out the "how" part. Also #822 is about improving type annotations, so I don't see how it is related.

Reply:
Should have been #828... Sorry. Give me a sec to clean up my mess